SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/Operations/SubJob/BrokerProfileOptOutSubJob.swift

```diff
@@ -47,6 +54,83 @@ struct BrokerProfileOptOutSubJob {
                           brokerProfileQueryData: BrokerProfileQueryData,
                           showWebView: Bool,
                           shouldRunNextStep: @escaping () -> Bool) async throws -> Bool {
+        // 1, 2, 3, 4. Validate function parameters and extract ID's
```

Comment:
I think we could remove these comments now tbh - I added them originally since we had a few things happening in sequence and it made them easier to follow, but now the functions speak for themselves. (The comments within the functions themselves could stay though.)